wifiphisher/common/recon.py

```diff
@@ -243,7 +243,7 @@ def _create_ap_with_info(self, packet):
 
         elt_section = packet[dot11.Dot11Elt]
         try:
-            channel = str(ord(packet[int(dot11.Dot11Elt):3].info))
+            channel = str(ord(packet[dot11.Dot11Elt:3].info))
```

Comment:
here should not cast to int since `Dot11Elt` is type of `<class 'scapy.layers.dot11.Dot11Elt'>`